.github/workflows/github-pages.yml

```diff
@@ -0,0 +1,29 @@
+# GitHub Action for Jekyll
+#
+# Runs jekyll build on master and pushes output to gh-pages branch.
+# (Then GitHub deploys that static content.)
+
+name: Build and deploy Jekyll site to GitHub Pages
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  github-pages:
+    runs-on: ubuntu-20.04
```

Comment:
I typically just stick with ubuntu-latest, as that keeps the project easier to maintain lightly over longer periods of time (e.g. 10+ years ;). Can we do that here?